src/sentry/integrations/bitbucket/webhook.py

```diff
@@ -104,9 +108,55 @@ def __call__(self, organization, event):
                     pass
 
 
+class PullEventWebhook(Webhook):
+    def __call__(self, organization, event):
+        repo = self.repo_from_event(organization, event)
+        pull_request = event['pullrequest']
+
+        integration = Integration.objects.get(
+            id=repo.integration_id,
+        )
+        installation = integration.get_installation(organization_id=organization.id)
+        emails = installation.get_client().get_user_emails()['values']
+        try:
+            commit_author = CommitAuthor.objects.filter(
+                organization_id=organization.id,
+                email__in=[email['email'] for email in emails if email['is_confirmed']],
+            )[0]
+        except IndexError:
+            primary_email = None
+            for email in emails:
+                if email['is_primary']:
+                    primary_email = email['email']
+                    break
+
+            commit_author = CommitAuthor.objects.create(
+                organization_id=organization.id,
+                email=primary_email,
+                name=pull_request['author']['display_name'],
```

Comment:
This can be None, is `CommitAuthor` name important? Should I try filling it in with something that is guarenteed to be there? I.E. username or something similar?

Reply:
can you look to see where it's used to determine whether there should be a backup?

Reply:
I'd +1 filling it with `username` if `display_name` isn't present, either way.